### PR TITLE
add README

### PR DESCRIPTION
**Pull request by will-moore**
Some notes from my discussion with @sbesson yesterday.
Any improvements appreciated!

Staged at https://github.com/will-moore/omero-plugins/tree/add_README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+omero-plugins
+=============
+
```
Comment (sbesson) [inline README.md]:
```suggestion

[![Build Status](https://travis-ci.org/ome/omero-plugins.svg)](https://travis-ci.org/ome/omero-plugins)

```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+This collection of OMERO plugins allows us to maintain a single cron job
+which runs travis daily for all the listed repositories (instead of
```
Comment (sbesson) [inline README.md]:
```suggestion
which runs Travis daily for all the listed repositories (instead of
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+maintaining separate cron jobs for each one). The results can be
+seen at https://travis-ci.org/ome/omero-plugins/builds. 
+
+Running travis daily for each repository allows us to check that they
```
Comment (sbesson) [inline README.md]:
```suggestion
Running Travis daily for each repository allows us to check that they
```

**Comment by will-moore** [comment]
@sbesson Thanks for the commits. Good to merge now?

**Pull request merged**